bin/rportal

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python3
+
+import argparse
+import os
+
+commands = {
+    "up": "docker-compose --env-file ./docker-compose.env up",
+    "down": "docker-compose --env-file ./docker-compose.env down",
+    "build": "docker-compose --env-file ./docker-compose.env build",
+    "rebuild-index": "docker-compose --env-file ./docker-compose.env run --rm api ./manage.py search_index --rebuild",
+    "populate-db": "docker-compose --env-file ./docker-compose.env run --rm api python3 manage.py populate_test_database",
+    "recreate-schema": "./recreate_schema.sh",
```

Comment:
`recreate_schema.sh` calls docker-compose, presumably now with the incorrect env file.

Reply:
Oh wait you changed that. I suppose it makes sense to keep recreate_schema the way it was because it needs to call docker-compose twice with different containers.